ush/prepobs_makeprepbufr.sh

```diff
@@ -924,10 +924,10 @@ fi
 #  determine local system name and type if available
 #  -------------------------------------------------
 SITE=${SITE:-""}
-sys_tp=${sys_tp:-$(getsystem.pl -tp)}
+sys_tp=${sys_tp:-$(getsystem -tp)}
```

Comment:
`getsystem -tp` is not valid on WC2. You need to use different options or none at all.